cg/store/user_filters.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum
+from typing import Callable, List, Optional
+from sqlalchemy.orm import Query
+from cg.store.models import User
+
+
+def filter_user_by_email(users: Query, email: str, **kwargs) -> Query:
+    """Return user by email."""
+    return users.filter(User.email == email)
+
+
+class UserFilter(Enum):
+    """Define User filter functions."""
+
+    FILTER_BY_EMAIL: Callable = filter_user_by_email
+
+
+def apply_user_filter(
+    users: Query,
+    filters: List[UserFilter],
```

Comment:
```suggestion
    filter_filters: List[Callable],
```
Did we not land on this approach in the stand-up this morning?